drivers/virtualbox/virtualbox.go

```diff
@@ -458,11 +458,6 @@ func (d *Driver) Start() error {
 		}
 	}
 
-	log.Infof("Waiting for an IP...")
-	if err := d.ipWaiter.Wait(d); err != nil {
-		return err
-	}
-
```

Comment:
This is not relevant here since at the end of `Start` method we are already waiting for the `IP` address.